chart/kubeapps/Chart.yaml

```diff
@@ -33,4 +33,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.6.2-dev0
+version: 7.6.2
```

Comment:
7.6.3-dev0 ?

Reply:
Ah, guess since we're not changing anything, 7.6.2 is what it is synced as... our first change in dev will make it 7.6.3-dev0 - sorry for the noise :)

Reply:
It depends... it's fine either way. Our chart will always differ from the bitnami one in the images. If we consider it is a change that leads to a `vX.Y.Z+1-dev0` it's ok. I'm also comfortable with leaving it as is (`vX.Y.Z`).

> I'm guessing this PR was automatically created as draft and meant to mark it as ready for review? Anyway, +1

They released this new version as part of a batch PR for solving an issue with their readmenator. However, they made some mistakes in the default value (using `""` instead of `{}`), but they will solve it soon. I didn't perform any changes to this PR as I was waiting for them to send the new one. That's why it is still a draft (this is how the bot creates the PRs).
So, my plan is to leave it open (because otherwise, the bot would create another one again and again) until they fix it, then close this one and perform the changes in the new PR.